docs/developer/architecture.md

```diff
@@ -0,0 +1,125 @@
+# Architecture
```

Comment:
A leading h1 is only necessary if you want to change the page title from that in the mkdocs.yml pages section. Since this is the same, it is superfluous and potentially confusing.

Reply:
oh thanks!